apps/testingaccessibility/src/pages/thanks/purchase.tsx

```diff
@@ -161,24 +184,23 @@ const ThanksVerify: React.FC<React.PropsWithChildren<ThanksProps>> = ({
               </>
             )}
             {!isNewPurchase && !!bulkCouponId && (
-              <>
-                <h2 className="max-w-lg mx-auto font-bold lg:text-4xl text-3xl py-5">
-                  Invite Your Team
-                </h2>
-                <div className="w-full text-gray-900">
-                  <CopyInviteLink bulkCouponId={bulkCouponId} />
-                </div>
-                <p className="text-sand-100 max-w-md font-medium leading-relaxed mx-auto mt-2">
-                  You can also visit your{' '}
-                  <Link href="/team/invite">
-                    <a className="py-1 inline-flex text-base font-medium hover:underline transition">
-                      Team Invite
-                    </a>
-                  </Link>{' '}
-                  page anytime to get the share link for distributing to your
-                  team.
-                </p>
-              </>
+              <div className="text-gray-900 mt-4">
+                <Card
+                  title={{as: 'h1', content: 'Invite your team'}}
+                  icon={
+                    <UserGroupIcon
+                      className="w-5 text-green-500"
+                      aria-hidden="true"
+                    />
+                  }
+                >
+                  <InviteTeam
+                    userEmail={email}
+                    purchase={bulkCouponPurchase}
+                    existingPurchaseForSelf={!!selfRedeemedPurchase}
+                  />
+                </Card>
+              </div>
```

Comment:
Here is the `<InviteTeam>` component being displayed on the post-purchase page.

For its default styles to make sense, I had to write it in a card. I also had to wrap a `<div>` around the whole thing with `text-gray-900` or else the text color was rendering as white.